packages/front-end/components/Metrics/MetricName.tsx

```diff
@@ -77,6 +77,11 @@ export function OfficialBadge({
                 <>
                   a <code>config.yml</code> file
                 </>
+              ) : managedBy === "admin" ? (
+                <>
+                  an Admin or someone with the{" "}
+                  <code>ManageOfficialResources</code> policy
```

Comment:
I don't know if this is the right place to put the policy name.  Might be better to just have this say `This metric is being managed by admins`.  Not 100% accurate in all cases, but easier to understand.